recipes/samarium/all/CMakeLists.txt

```
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.1)
+project(cmake_wrapper)
+
+list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR})
+list(APPEND CMAKE_PREFIX_PATH ${CMAKE_CURRENT_SOURCE_DIR})
```

```suggestion
include(conanbuildinfo.cmake)
conan_basic_setup(KEEP_RPATHS)
```

Need to load conan_basic_setup to inject bin folder, compiler flags, etc ..